pom.xml

```diff
@@ -0,0 +1,351 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.ibmix.magkit</groupId>
+    <artifactId>magkit-pom</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <name>IBM iX Magkit-POM</name>
+    <description>IBM iX Magkit POM which should be used as the top level pom of the project structure for iX-Magkit projects. This POM
+        defines the common management sections with their default settings for dependencies and plugins.
+    </description>
+    <url>https://github.com/IBM/magkit-pom</url>
+    <inceptionYear>2023</inceptionYear>
+
+    <organization>
+        <name>IBM iX</name>
+        <url>https://ibmix.de</url>
+    </organization>
+
+    <properties>
+        <!-- Magnolia version, override this to use another version -->
+        <magnoliaVersion>6.2.33</magnoliaVersion>
+        <magnolia.content-types.version>1.2</magnolia.content-types.version>
+        <magnolia.site.version>1.4</magnolia.site.version>
+
+        <source.javaversion>11</source.javaversion>
+        <target.javaversion>11</target.javaversion>
+
+        <source.encoding>UTF-8</source.encoding>
+        <project.build.sourceEncoding>${source.encoding}</project.build.sourceEncoding>
+
+        <!-- checkstyle can be disabled with this property -->
+        <checkstyle.skip>false</checkstyle.skip>
+        <checkstyle.config>src/config</checkstyle.config>
+        <checkstyle.excludes>**/package-info.java</checkstyle.excludes>
+        <maven.checkstyle.plugin>3.1.2</maven.checkstyle.plugin>
```

Comment:
use allways same naming convention for version properties, e.g.
`<version.module-name>1.0.0<version.module-name>`

Reply:
done!